schema/caveschema.xsd

```diff
@@ -29,10 +28,109 @@
             </xs:sequence>
           </xs:complexType>
         </xs:element>
-        <xs:element name="PlacementRoot" minOccurs="0" maxOccurs="1">
+        <xs:element name="PlacementRoot" maxOccurs="1">
           <xs:complexType>
             <xs:sequence>
-              <xs:element ref="Placement" maxOccurs="unbounded" />
+              <!-- Center -->
+              <xs:element name="Placement">
```

Comment:
should this be called "Placement" rather than "Center"? Same applies on the other elements "Front Wall", "Floor", "Right Wall"...

Reply:
I was more referring to the name of the Placement itself... I could have the comment be the xml tag?